management/server/user.go

```diff
@@ -270,9 +296,11 @@ func (am *DefaultAccountManager) inviteNewUser(accountID, userID string, invite
 
 	role := StrRoleToUserRole(invite.Role)
 	newUser := &User{
-		Id:         idpUser.ID,
-		Role:       role,
-		AutoGroups: invite.AutoGroups,
+		Id:                   idpUser.ID,
+		Role:                 role,
+		AutoGroups:           invite.AutoGroups,
+		Issued:               invite.Issued,
+		IntegrationReference: invite.IntegrationReference,
```

Comment:
In case of CreateUser **user.go:236** API endpoint the UserInfo does not contain the IntegrationReference but here you add it to the User structure. Is it correct?